src/components/wallet/methods/claimAsset.ts

```diff
@@ -7,23 +7,23 @@ import {
   Keypair,
 } from 'stellar-sdk'
 import { handleError } from '@services/error'
+import { ClaimableBalance } from '../views/claimableDisplay'
 import { Wallet } from '../wallet'
 
 export default async function claimAsset(
   this: Wallet,
-  balanceId: string,
-  assetCode: string,
-  sponsor: string,
-  amount: string
+  balance: ClaimableBalance
 ) {
-  const loadingKey = `claim:${assetCode}:${balanceId}`
+  const balanceId = balance.id
```

Comment:
Is this variable used? I see you use `balance.id` everywhere else

Reply:
For some reason ts was complaining when I used `balance.id` here `Operation.claimClaimableBalance`.
So I assigned it to a new variable `balanceId` where its called on line 44
```
      .addOperation(
        Operation.claimClaimableBalance({
          balanceId,
        })
      )
```

Reply:
Oh I see one usage. Ok! I'm going to merge this

Reply:
Ohh I bet it was because `claimClaimableBalance` expects an object with `balanceId` as a key. The `{ key }` shorthand syntax is short for: `{ key: value }` but it only works when the variable name matches the key name.